extensions/amp-story/1.0/amp-story.js

```diff
@@ -1841,6 +1847,13 @@ export class AmpStory extends AMP.BaseElement {
         break;
       case UIType.DESKTOP_ONE_PANEL:
         this.setDesktopPositionAttributes_(this.activePage_);
+        if (!this.backgroundBlur_) {
+          this.backgroundBlur_ = new AmpStoryBackgroundBlur(
```

Comment:
Should we set it to `null` when switching to another UIType?

Reply:
Good call. I added a `detach` method to remove it from the dom before setting it to null.